migrations/1695119925617_add-trgm-extension.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.addExtension('pg_trgm', { ifNotExists: true })
```

Comment:
I've tried before to install extensions using migrations with no luck. It is pretty possible that this will fail when deploying the service in our infra so it might be better to do it manually.